dissect/cstruct/types/base.py

```diff
@@ -181,6 +181,15 @@ def _write_0(cls, stream: BinaryIO, array: list[BaseType]) -> int:
         """
         return cls._write_array(stream, array + [cls()])
 
+    def _class_stub(cls) -> str:
+        return f"class {cls.__name__}({cls.__base__.__name__}):\n"
+
+    def _type_stub(cls, name: str = "") -> str:
+        return f"{name}: {cls.__name__}"
+
+    def to_stub(cls, name: str) -> str:
```

Comment:
Maybe name this differently? Also I think this might as well be a private method. Maybe `_to_type_stub`? `to_stub` might be confusing because it also makes me think of the compiler.

Reply:
I don't agree on the "externally" part, I'd classify this usage as internal and would never expect anyone to ever call this externally. This is not part of any expected usage API.